tests/RepositoryForms/FieldType/Mapper/UserAccountFieldValueFormMapperTest.php

```diff
@@ -19,6 +20,15 @@ protected function setUp()
         parent::setUp();
 
         $formRoot = $this->getMockBuilder(FormInterface::class)->getMock();
+        $formRoot->expects($this->once())
```

Comment:
I'm against using `expects` method in mocked objects. If you indeed want to check that the method is called x times it alright but in cases where method can be called multiple times (safe for getters) it's a inconvenient since small change like calling `getData()` twice will fail the test...